packages/babel-helper-transform-fixture-test-runner/package.json

```diff
@@ -20,7 +20,6 @@
     "@babel/helper-fixtures": "workspace:^7.13.10",
     "babel-check-duplicated-nodes": "^1.0.0",
     "escape-string-regexp": "condition:BABEL_8_BREAKING ? ^4.0.0 : ",
```

Comment:
```suggestion
```
Can remove this as well now since you removed the file that uses it

Reply:
good catch thanks!!